pyscriptjs/tests/integration/test_zz_examples.py

```diff
@@ -276,18 +276,20 @@ def test_panel_deckgl(self):
         self.check_tutor_generated_code()
 
     def test_panel_kmeans(self):
-        # XXX improve this test
+        # XXX improve this test>>>>>>> main
         self.goto("examples/panel_kmeans.html")
-        self.wait_for_pyscript(timeout=90 * 1000)
+        self.wait_for_pyscript(timeout=120 * 1000)
```

Comment:
Terrible. What kind of test requires a twelve second timeout? Scipy is the worst.

It'll be really nice once jspi is shipping in browsers and we don't have to preload scipy's 100+ .so files. In the meantime I think we should get rid of kmeans. If a single test doesn't test anything and requires a twelve second timeout it needs to be gotten rid of. If it does test something can anyone tell me what it tests and why it can't be done in a scipy free way?

Reply:
@hoodmane for what it's worth, that's a _120 second_ timeout 🤢

I'm more than happy to go an entirely different direction with testing and removing the examples from the testing workflow, or testing them only prior to release, or what have you. I was mostly looking for a straightforward path to get this long-standing PR merged and the project moved up to Pyodide 0.23/Python 3.11.

How would we feel about merging this as-is and following up with a separate PR/discussion about improving the examples testing process?

Reply:
that would work for me

Reply:
Rather than only before the release, I'd personally prefer to run them nightly or weekly or something like that. At least we are aware something broke sooner (and we don't have the overhead of testing examples before merging a PR)

In terms of testing too much 3rd party, I agree that we should test less and mainly focus on "if they load, we are good". There are some caveats though that I'd like for us to consider and add better support to specific 3rd party libraries. That's especially true for modules that are quite popular and don't just work with specifying them as dependencies. Take `panel` or `bokeh` as examples for instance, I'd like for us to make the user experience better and "auto-patch" everything they need to "just work" when you specify them as packages (but that is a totally different discussion and I think it probably belongs to PyScript core and rather to higher level API or something like this)

Reply:
> I'd personally prefer to run them nightly or weekly or something like that

that's a great idea too, as long as these are off our daily pipes around unrelated tasks